src/jit/compiler.h

```diff
@@ -1133,6 +1133,7 @@ struct FuncInfoDsc
                                // funclet. It is only valid if funKind field indicates this is a
                                // EH-related funclet: FUNC_HANDLER or FUNC_FILTER
 
+#if FEATURE_EH_FUNCLETS
 #if defined(_TARGET_AMD64_)
```

Comment:
It looks like you've added `#if FEATURE_EH_FUNCLETS`, but x86 won't get any of the enclosed declarations since there is no `_TARGET_X86_` section. So can you remove the (unnecessary) added `#if FEATURE_EH_FUNCLETS`?